### PR TITLE
Do not pluralize resource name on new page

### LINKED ISSUE #1205: New page title should not be pluralized
This:
https://github.com/thoughtbot/administrate/blob/2c54226aeaebc1ac5316ccd22e87d3080e3608ed/app/views/administrate/application/new.html.erb#L21

Should probably use the same logic as this:
https://github.com/thoughtbot/administrate/blob/2c54226aeaebc1ac5316ccd22e87d3080e3608ed/app/views/administrate/application/index.html.erb#L47

### PR DESCRIPTION
**Pull request by dvanderbeek**
fixes #1205

### CODE REVIEW

File: app/views/administrate/application/new.html.erb [hunk 1]
```diff
@@ -18,7 +18,7 @@ to do the heavy lifting.
 <% content_for(:title) do %>
   <%= t(
     "administrate.actions.new_resource",
-    name: display_resource_name(page.resource_name).titleize
+    name: page.resource_name.titleize
```
Comment (nickcharlton) [inline app/views/administrate/application/new.html.erb]: Is there a corresponding test for this, too? If not, would you be able to contribute one?

File: app/assets/stylesheets/administrate/base/_forms.scss [hunk 2]
```diff
@@ -79,6 +79,7 @@ textarea {
 }
 
 textarea {
+  min-height: 550px;
```
Comment (nickcharlton) [inline app/assets/stylesheets/administrate/base/_forms.scss]: Do you mean to change this in this PR too?

Comment (dvanderbeek) [reply app/assets/stylesheets/administrate/base/_forms.scss]: oops, no I only meant to do that in my fork which I have an app using directly from github. I'll revert this change as soon as I can.

File: app/views/administrate/application/new.html.erb (hunk 1 above)
Comment (dvanderbeek) [reply app/views/administrate/application/new.html.erb]: Will do

Comment (vifreefly) [reply app/views/administrate/application/new.html.erb]: Looks like it didn't work with i18n translated model name

**Comment by dvanderbeek** [comment]
@nickcharlton I finally had a chance to clean up this PR. Let me know how it looks!

**Pull request closed without merging**